steps/lambdatest-upload/0.0.1/step.yml

```diff
@@ -0,0 +1,80 @@
+title: lambdatest-upload
+summary: |
+  Upload APK/IPA files to LambdaTest's App Automate
+description: |
+  Upload your build files to Lambdatest for remote device testing. Upload your IPA/APK file to Lambdatest App Automation and get back the app's url for your remote device tests
+website: https://github.com/LambdaTest/bitrise-step-lambdatest-upload
+source_code_url: https://github.com/LambdaTest/bitrise-step-lambdatest-upload
+support_url: https://github.com/LambdaTest/bitrise-step-lambdatest-upload/issues
+published_at: 2022-04-01T11:46:49.57948+05:30
+source:
+  git: https://github.com/LambdaTest/bitrise-step-lambdatest-upload.git
+  commit: 7cd2ff643722f2b89aa12fb2e3bd32a6ff60f56d
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- ios
+- macos
+- android
+- xamarin
+- react-native
+- cordova
+- ionic
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: git
+  - name: wget
+  - name: jq
+  - name: ack
+  apt_get:
+  - name: git
+  - name: wget
+  - name: jq
+  - name: ack
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- opts:
+    description: |
+      This usually gets generated by the the "Gradle Runner" or "XCode Archive & Export" step
+    is_required: true
+    summary: The app file you want to upload to LambdaTest, usually $BITRISE\_APK\_PATH
```

Comment:
You might want to use backticks `` to highlight `$BITRISE_APK_PATH` and `$BITRISE_IPA_PATH` environment variables